onebusaway-gtfs-transformer/src/main/java/org/onebusaway/gtfs_transformer/services/AwsContextService.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2018 Cambridge Systematics, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.onebusaway.gtfs_transformer.services;
+
+import org.onebusaway.gtfs.model.Agency;
+import org.onebusaway.gtfs.services.GtfsMutableRelationalDao;
+
+public class AwsContextService {
```

Comment:
I like the pattern you've created, but I have to object to naming it Aws...
onebusway-gtfs-modules should know nothing of the cloud portion of it, hence why we created onebusaway-cloud-services.  This might be more aptly named FeedContextServer.

Reply:
Also consider CloudContextService, to handle the fact you have a getNamespace method...